.github/workflows/run-onecc-build.yml

```diff
@@ -0,0 +1,98 @@
+name: Run ONECC Ubuntu Build
```

Comment:
maybe ?
```suggestion
name: Run ONECC Ubuntu 20.04 Build
```

Reply:
This workflow can support many ubuntu version. So I'll try to add version in job name, not workflow name.